engine_graphics/src/Renderer.cpp

```diff
@@ -0,0 +1,463 @@
+#include <GL/glew.h> //beacause of some cryptic shit this motherfucker has to be the first
+#include <core/Renderer.hpp>
+
+#include <core/gl_extensions.hpp>
+#include <GL/freeglut.h>
+
+#include <deque>
+
+namespace rcbe::core::rendering
+{
+class RendererImplementation
+{
+public:
+    using handle_type = int;
```

Comment:
Since handle is not used outside this class, this is obsolete.

Reply:
This might be used later, leaving it be.